bootstrap.sh

```diff
@@ -27,9 +37,14 @@ for file in $FILES; do
             echo "~/"$file" does not exist."
             echo "Create link to "$file"? [y/n]"
         fi
-        read -p "> " confirm && [[ $confirm == [yY] || $confirm == [yY][eE][sS] ]] || continue
+        # The "-n" option for `read` will limit user input to 1 character. I
+        # like the feel of this better
+        read -p "> " -n 1 confirm && [[ $confirm == [yY] ]] || continue
     fi
     if [[ $FORCE = true || $INTERACTIVE = true ]]; then
+        # the rm spooks me. I'm stupid, so I'd get burned if I left this as is
+        # on my machine lol. However, not just rm'ing it makes it more
+        # complicated so I also respect the simplicity.
```

Comment:
I figured explicitly adding an option to running the script  and/or typing 'yes'  is enough for me to think about it